components/bluetooth/lib.rs

```diff
@@ -1093,4 +1093,11 @@ impl BluetoothManager {
             None => return drop(sender.send(Err(BluetoothError::InvalidState))),
         }
     }
+
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothdevice-watchadvertisements
+    fn watch_advertisements(&mut self, _device_id: String, sender: IpcSender<BluetoothResponseResult>) {
+        // TODO: Implement this when supported in lower level
```

Comment:
I know but there are identical cases in the code with this solution.
https://github.com/servo/servo/blob/22aebdf5d41a3509cd6515ccf5edcdf33715a76d/components/script/dom/request.rs#L94
https://github.com/servo/servo/blob/72e4c6dc21c132d49cf0a5f68a3ba45d16cc8322/components/script/dom/htmlanchorelement.rs#L458